### PR TITLE
proper xml response in case of error #37

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues
#37 

Thanks for contributing!

### CODE REVIEW

File: src/status.js [hunk 1]
```diff
@@ -15,11 +15,27 @@ const { version } = require('../package.json');
...
-  ]);
+  try {
+    await Promise.all([
+      request.get('https://api.fastly.com/docs'),
+      request.get('https://iam.googleapis.com/$discovery/rest?version=v1'),
+      request.get('https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest'),
+    ]);
+  } catch (err) {
+    return {
+      statusCode: 200,
```
Comment (trieloff) [inline src/status.js]: Shouldn't this be error 503?

**Comment by trieloff** [comment]
We could also just use the new `helix-pingdom-status` Project

File: src/status.js (hunk 1 above)
Comment (stefan-guggisberg) [reply src/status.js]: agreed

Comment (stefan-guggisberg) [reply src/status.js]: agreed

**Comment by stefan-guggisberg** [comment]
> We could also just use the new `helix-pingdom-status` Project
see #40

**Pull request merged**